eva/parser/create_udf_statement.py

```diff
@@ -93,7 +93,10 @@ def __str__(self) -> str:
                 metadata_str += f"{key}={value}, "
             metadata_str = metadata_str.rstrip(", ")
 
-        return f"CREATE UDF {self._name} INPUT ({input_str}) OUTPUT ({output_str}) TYPE {self._udf_type} IMPL {self._impl_path.name} ({metadata_str}))"
+        if self._impl_path:
+            return f"CREATE UDF {self._name} INPUT ({input_str}) OUTPUT ({output_str}) TYPE {self._udf_type} IMPL {self._impl_path.name} ({metadata_str}))"
```

Comment:
There is no need to print `INPUT` and `OUTPUT.` We are anyways getting rid of it.